resources/install/protected/update_1.4.sql

```diff
@@ -58,7 +58,7 @@ CREATE TABLE IF NOT EXISTS `Downtime` (
   `creatorID` varchar(80) NOT NULL,
   `dateCreated` datetime NOT NULL,
   `startDate` datetime NOT NULL,
-  `endDate` datetime NOT NULL,
+  `endDate` datetime DEFAULT NULL,
```

Comment:
Thanks for adding this to the 1.41 file. Could you remove it from update_1.4.sql?

Please also change resources/install/protected/install.sql to match. It currently has `'endDate' datetime NOT NULL`.